<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptDisplayDlg</class>
 <widget class="QDialog" name="OptDisplayDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="Q3GroupBox" name="GroupBox17">
     <property name="title">
      <string>Grid</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="pb_gridcolor">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_gridcolor">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="TextLabel14">
          <property name="text">
           <string>Size:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_gridsize">
          <property name="minimum">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox3">
     <property name="title">
      <string>Shadows</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="cb_shadows">
          <property name="text">
           <string>Display shadows</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="pb_shadowcolor">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lb_shadowcolor">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3ButtonGroup" name="bg_transitions">
     <property name="title">
      <string>Transitions</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="cb_iomark">
          <property name="text">
           <string>Use '?' and '!' to mark inputs and outputs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_ionames">
          <property name="text">
           <string>Display input/output names</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_drawbox">
          <property name="text">
           <string>Draw Box around inputs/outputs</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_tooltips">
     <property name="text">
      <string>Tooltips</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_start"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>Default transition descriptor:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Inversion descriptor:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_default"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lb_start">
       <property name="text">
        <string>Initial transition descriptor:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_any"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_inversion"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel1_2">
       <property name="text">
        <string>&quot;Any input&quot; descriptor:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pb_gridcolor</tabstop>
  <tabstop>sb_gridsize</tabstop>
  <tabstop>pb_shadowcolor</tabstop>
  <tabstop>cb_iomark</tabstop>
  <tabstop>cb_ionames</tabstop>
  <tabstop>cb_drawbox</tabstop>
  <tabstop>cb_tooltips</tabstop>
  <tabstop>le_start</tabstop>
  <tabstop>le_inversion</tabstop>
  <tabstop>le_any</tabstop>
  <tabstop>le_default</tabstop>
  <tabstop>cb_shadows</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_gridcolor</sender>
   <signal>clicked()</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>chooseGridColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_tooltips</sender>
   <signal>clicked()</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>tooltipsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_start</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>startDescChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_iomark</sender>
   <signal>clicked()</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>ioMarkClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_shadows</sender>
   <signal>clicked()</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>shadowsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_shadowcolor</sender>
   <signal>clicked()</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>chooseShadowColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_drawbox</sender>
   <signal>clicked()</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>drawBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_ionames</sender>
   <signal>clicked()</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>ioNamesClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_inversion</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>inversionDescChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_any</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>anyInputDescChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_default</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OptDisplayDlg</receiver>
   <slot>defaultDescChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
